src/modules/common/models/SelectorItemModel.js

```diff
@@ -1,8 +1,8 @@
 export default class SelectorItemModel {
```

Comment:
Not related to this PR, but SelectorItemModel is no model it's just a SelectorItem and should be in the root of the common/ module.

Reply:
Models are domain-specific objects which come from the backend.

Reply:
okay